new/rfc-for-hugs.md

```diff
@@ -0,0 +1,51 @@
+---
+RFC: unassigned
+Author: Aaron Kalin and JJ Asghar
+Status: Draft
+Type: Informational, Process
+---
+
+# Hugging in the Chef Community
+
+In order to bridge the cultural gap between the US and Worldwide community the concept of hugging should be explained.
+
+## Motivation
+
+    As a Chef Community Participant,
+    I want to express positive emotion to or for another Chef Community Participant,
+    so that we can express our friendship physically.
+
+## Specification
+
+When approaching another Chef Community Member that you wish to engage in a hug. To comply with the already
+existing Chef Community Guidelines and Code of Conduct at various community events, you must seek permission from the
+other community member by either verbally or physically indicating that you wish to hug them.
```

Comment:
1) The first sentence ends abruptly. Editing error?

2) We should cite [RFC020](https://github.com/chef/chef-rfc/blob/master/rfc020-community-guidelines.md) and the chefconf code of conduct. maybe the latter as an example, because code of conducts vary between events.

3) The tone of "you must seek permission" combined with "to comply [with the rules]" doesn't feel right to me. 

For 1+2+3, I'll suggest:

Foremost, we want all community members to feel safe and welcome. This is expressed in the Chef Community Guidelines ([RFC020](https://github.com/chef/chef-rfc/blob/master/rfc020-community-guidelines.md)) and the Code of Conduct for our events (e.g. [ChefConf](https://www.chef.io/blog/chefconfcodeofconduct/)). When approaching another Chef Community Member that you wish to engage in a hug, you must first seek permission from the other community member by either verbally or physically indicating that you wish to hug them.